### PR TITLE
Backporting RUST updates from Bay.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
Ready to go.

### CODE REVIEW

File: code/modules/power/fusion/consoles/gyrotron_control.dm [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/obj/machinery/computer/fusion/gyrotron
+	name = "gyrotron control console"
+	icon_keyboard = "med_key"
+	icon_screen = "gyrotron_screen"
+	light_color = COLOR_BLUE
```
Comment (Lohikar) [inline code/modules/power/fusion/consoles/gyrotron_control.dm]: `LIGHT_COLOR_BLUE`?

File: code/modules/power/fusion/gyrotron/kinetic_harvester.dm [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+	var/obj/machinery/power/fusion_core/harvest_from
+
+/obj/machinery/kinetic_harvester/Initialize()
+	set_extension(src, /datum/extension/fusion_plant_member, /datum/extension/fusion_plant_member)
+	if(initial_id_tag)
+		var/datum/extension/fusion_plant_member/fusion = get_extension(src, /datum/extension/fusion_plant_member)
+		fusion.set_tag(null, initial_id_tag)
+	find_core()
+	queue_icon_update()
+	. = ..()
```
Comment (Lohikar) [inline code/modules/power/fusion/gyrotron/kinetic_harvester.dm]: This one should be moved too.

File: code/modules/power/fusion/consoles/gyrotron_control.dm (hunk 1 above)
Comment (MistakeNot4892) [reply code/modules/power/fusion/consoles/gyrotron_control.dm]: Whyfor?

File: code/modules/power/fusion/consoles/_consoles.dm [hunk 3]
```diff
@@ -0,0 +1,55 @@
+/obj/machinery/computer/fusion
+	icon_keyboard = "power_key"
+	icon_screen = "rust_screen"
+	light_color = COLOR_ORANGE
```
[the remark this answers is not in the document]
Comment (MistakeNot4892) [reply code/modules/power/fusion/consoles/_consoles.dm]: No idea. This is largely copypasted.

File: code/modules/power/fusion/consoles/gyrotron_control.dm (hunk 1 above)
Comment (MistakeNot4892) [reply code/modules/power/fusion/consoles/gyrotron_control.dm]: Resolved on Discord.

**Pull request merged**